chall04/isoria-g.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/python3
+
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    isoria-g.py                                        :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: marvin <marvin@student.42.fr>              +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/04/13 21:23:38 by marvin            #+#    #+#              #
+#    Updated: 2020/04/13 21:23:38 by marvin           ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
+
+import sys
+import os
+
+def findnum():
+    totalsize = 0
+    for item in sizes:
+        totalsize = totalsize + item
+    return (totalsize // maxshelf + 1)
```

Comment:
I understand your logic. But it doesn't makes sense as you only have access to one of each shelf. To illustrate this, take this output:
```
500
500 book1
500 book2
```
Your program returns that I need to shelves, but there only is one available in the store.  
Also with this input:
```
500 200 100
500 book1
300 book2
```
Your program returns that I need 2 shelves, but I only have one shelf of 500 and one of 200 and one of 100, the 2 lasts can't hold a book of 300 as I can not merge them.

Both examples should return an error.